tests/e2e/system_test/test_graceful_nodes_shutdown.py

```diff
@@ -0,0 +1,419 @@
+"""
+Test to verify that data is accessible and uncorrupted as well as
+operational cluster after graceful nodes shutdown
+"""
+
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.ocp import OCP, wait_for_cluster_connectivity
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.longevity import start_ocp_workload
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.registry import check_if_registry_stack_exists
+from ocs_ci.ocs.monitoring import check_if_monitoring_stack_exists
+from ocs_ci.framework import config
+from ocs_ci.ocs.uninstall import (
+    remove_ocp_registry_from_ocs,
+    remove_monitoring_stack_from_ocs,
+)
+from ocs_ci.utility.uninstall_openshift_logging import uninstall_cluster_logging
+from ocs_ci.framework.pytest_customization.marks import (
+    system_test,
+    polarion_id,
+    skipif_no_kms,
+    skipif_ocs_version,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs.node import (
+    get_nodes,
+)
+from ocs_ci.ocs.resources.fips import check_fips_enabled
+
+logger = logging.getLogger(__name__)
+
+
+class TestGracefulNodesShutdown(E2ETest):
+    """
+    Test uncorrupted data and operational cluster after graceful nodes shutdown
+    """
+
+    bucket_names_list = []
+
+    @pytest.fixture(autouse=True)
+    def checks(self):
+        # This test is skipped due to https://issues.redhat.com/browse/ENTMQST-3422
+
+        try:
+            check_fips_enabled()
+        except Exception as e:
+            logger.info(f"Handled prometheuous pod exception {e}")
+
+        """
+        nodes = get_nodes()
+        for node in nodes:
+            assert (
+                node.get()["status"]["allocatable"]["hugepages-2Mi"] == "64Mi"
+            ), f"Huge pages is not applied on {node.name}"
+        """
```

Comment:
If this code is not needed, please remove it

Reply:
It is needed. On testing cluster it was not enabled so for testing purpose its commented.